Cargo.toml

```diff
@@ -23,15 +23,17 @@ use-serde = ["serde", "bitcoin_hashes/serde"]
 bech32 = "0.7.1"
 byteorder = "1.2"
 bitcoin_hashes = "0.7"
-bitcoinconsensus = { version = "0.16", optional = true }
-secp256k1 = "0.12"
+bitcoinconsensus = { version = "0.17", optional = true }
 serde = { version = "1", optional = true }
 
 [dependencies.hex]
 version = "=0.3.2"
 
+[dependencies.secp256k1]
```

Comment:
I feel like someone talked about it somewhere but can't remember where.
But I personally prefer `secp256k1 = { version = "0.15", features = ["rand"] }` over this multi line thing (except in a case of a very long line).
that way changes to the dependency are clear and affect everything related to that dep

Reply:
agree, amended